interactions/client/bot.py

```diff
@@ -1497,32 +1554,24 @@ def __new__(cls, client: Client, *args, **kwargs) -> "Extension":
 
         client._extensions[cls.__name__] = self
 
+        if client._websocket.ready.is_set() and client._automate_sync:
+            client._loop.create_task(client._Client__sync())
+
         return self
 
-    def teardown(self):
+    async def teardown(self):
```

Comment:
Isn't this breaking? Feels like that may not be the best idea here.

Reply:
it's already in unstable lol, i had it already in the last pr

I have no clue why GitHub counts it as new addition

Reply:
Breaking yes, though I believe that this branch isn't completely in sync with v4.2.0;
On stable, this is already async (since #776)

Reply:
Yeah there's quite a bit of uh Git mess here lol